dcmdump/src/main.rs

```diff
@@ -40,6 +42,7 @@ where
     Str(T),
     DateTime(T),
     Invalid(T),
+    Nothing(T),
```

Comment:
I was thinking more along the lines of this. If there really is no value to dump, we don't need to take a value.

```suggestion
    Nothing,
```